packages/scss/src/components/inputs/textfield/_textfield.compact.scss

```diff
@@ -150,12 +91,6 @@
 	}
 
 	&.mod-autocomplete, &.mod-select, &.mod-search {
```

Comment:
We should be able to get rid of any display mods in this stylesheet as it is similar to the textfield classic

Reply:
✔️